packages/kit/src/internal/cjs.ts

```diff
@@ -15,106 +16,133 @@ export interface ResolveModuleOptions {
 export interface RequireModuleOptions extends ResolveModuleOptions {
   // TODO: use create-require for jest environment
   // native?: boolean
-  /** Clear the require cache (force fresh require) but only if not within `node_modules` */
+  /**
+   * Clear the require cache (force fresh require)
+   * but only if not within `node_modules`
+   */
   clearCache?: boolean
 
   /** Automatically de-default the result of requiring the module. */
   interopDefault?: boolean
 }
 
+// eslint-disable-next-line jsdoc/require-param, jsdoc/require-returns
 /** @deprecated Do not use CJS utils */
-function isNodeModules (id: string) {
+function isNodeModules(id: string) {
```

Comment:
can we keep the space before brackets? 🙏

Reply:
Done :)